pandas/core/generic.py

```diff
@@ -6815,16 +6815,21 @@ def mask(self, cond, other=np.nan, inplace=False, axis=None, level=None,
                           errors=errors)
 
     _shared_docs['shift'] = ("""
-        Shift index by desired number of periods with an optional time freq
+        Shift index by desired number of periods with an optional time freq.
```

Comment:
References to parameters in the descriptions / notes should be put in backticks, so \'freq\' wherever applicable (couple of other instances below)